src/services/navigationBar.ts

```diff
@@ -700,4 +703,16 @@ namespace ts.NavigationBar {
                 return false;
         }
     }
+
+    function cleanText(text: string): string {

```

Comment:
Even if we're not going to call this in a single location, can we assert in a single location that the navbar item text never contains newlines?

Reply:
I feel like checking again should be solved by the tests instead of asserting. I have updated an existing test and added a couple more for avoiding introducing the same issues.